### ISSUE TITLE
Simple vs. advanced UI should be a client-side change

### ISSUE THREAD

**Issue by trwnh** [tootsuite/mastodon#10851]
I have multiple devices and want some to be using simple mode (e.g. a laptop) and others to use advanced mode (e.g. a desktop with a 2560px-wide monitor).

Having the setting "enable advanced web UI" be in preferences rather than a client-side toggle does not allow this.

**Comment by Gargron** [comment]
Yeah but in the same vein, which columns are pinned, or which notifications are hidden, and other web UI settings could be desired to be different between browser sessions. But that is not what we have, and it makes sense to, because if you lost all your customizations when you logged out, or if you had to re-set all your needed settings on each new machine it would suck. The layout type is no different.

**Comment by trwnh** [comment]
it might be a minor hassle to have to configure certain things on certain browsers, but it also sucks more that having different configs is literally impossible. i probably switch devices more than most people, and i'd be willing to configure the webapp 3 times a year (when i start using new devices) as opposed to 12 times a day (when i switch between my existing devices)

technically several of the toggles under "preferences" are not account preferences, but rather web-app preferences; these preferences that target only the web-app should be grouped together and kept in localStorage imo

**Comment by Gargron** [comment]
I'm not sure what can be done. I'm not ready to give up existing functionality. Maybe a query string param could override the saved setting for what layout to use? But then again, nobody will know about that, adding something for just one person doesn't make sense either.

**Comment by realityfabric** [comment]
There could be an option to sync settings across devices (as it does), which could be disabled by device and stored in local app settings. Storing each device's config on the server seems cumbersome,  but would it be possible to have "profiles"? Similar to how Firefox/Chrome let you have different configs and switch between them?

This would be a cool feature, but I do wonder how many people would actually use it.

**Comment by jeroenpraat** [comment]
A simple toggle link in the 'getting started' (advanced) / left sidebar (simple) footer (so that needs a query string param I suppose) would save a lot of headaches.

**Comment by trwnh** [comment]
Perhaps have the checkbox in settings be the "default" across sessions, and then re-introduce the React toggle for local overrides?

**Comment by trwnh** [comment]
would still like to see this.
